tests/conftest.py

```diff
@@ -3976,7 +3979,7 @@ def reportportal_customization(request):
 
 
 @pytest.fixture()
-def multi_pvc_clone_factory(pvc_clone_factory):
+def multi_pvc_clone_factory(pvc_clone_factory, pod_factory):
```

Comment:
This change can cause the existing calls to this function to fail

Reply:
I don't think the function would fail because while calling a fixture we do not pass the list of factories as arguments, we only pass the arguments mentioned in the def factory func, and all those are optional.